vue-components/src/components/OptionsMenu.vue

```diff
@@ -106,6 +133,17 @@ export default Vue.extend( {
 					break;
 			}
 		},
+		keyboardScroll(): void {
+			const element = this.$refs[ 'menu-items' ] as HTMLElement[];
+
+			if ( this.scrollWithKeyboard && this.keyboardHoveredItemIndex != -1 ) {
+				element[ this.keyboardHoveredItemIndex ].scrollIntoView( {
```

Comment:
And once you have removed the prop, I think you can fix the tests with this approach: https://stackoverflow.com/a/52446309/3293343